padd.sh

```diff
@@ -105,28 +105,40 @@ TestAPIAvailability() {
 
     local chaos_api_list authResponse cmdResult digReturnCode authStatus authData apiAvailable
 
-    # Query the API URLs from FTL using CHAOS TXT
-    # The result is a space-separated enumeration of full URLs
-    # e.g., "http://localhost:80/api" or "https://domain.com:443/api"
-    if [ -z "${SERVER}" ] || [ "${SERVER}" = "localhost" ] || [ "${SERVER}" = "127.0.0.1" ]; then
-        # --server was not set or set to local, assuming we're running locally
-        cmdResult="$(dig +short chaos txt local.api.ftl @localhost 2>&1; echo $?)"
-    else
-        # --server was set, try to get response from there
-        cmdResult="$(dig +short chaos txt domain.api.ftl @"${SERVER}" 2>&1; echo $?)"
+    if [ -n "$API_LOCATION" ] && [ -n "$SERVER" ]; then
```

Comment:
Could you please move the check for set flags down to where we set the options. No need to enter any function if we know we're going to exit anyway.